app/lib/frontend/oauth2_service.dart

```diff
@@ -36,9 +40,11 @@ class OAuth2Service {
     try {
       final api = new oauth2_v2.Oauth2Api(authClient);
       info = await api.userinfo.get();
+    } catch (e, st) {
+      _logger.log(Level.INFO, 'Bearer token lookup failed.', e, st);
```

Comment:
Can we catch a known exception, such as access denied? Then we can log that as info, while unknown exceptions can be logged as errors / warnings?